### ISSUE TITLE
Mastodon icon appearing black in WNP's dark mode

### ISSUE THREAD

**Issue by reemhamz** [mozilla/bedrock#12569]
## Description
There is an oopsie happening in WNP's dark mode footer:
<img width="230" alt="Screen Shot 2023-01-06 at 4 40 28 PM" src="https://user-images.githubusercontent.com/42309026/211104661-fa3dad17-a1b1-4968-b015-1bf3347c2a40.png">

### Steps to reproduce

Go to the What's New page: https://www.mozilla.org/en-US/firefox/whatsnew/
and scroll all the way down to the footer.

### Expected result
The Mastodon icon should be white!

### Actual result
The icon is black! **In dark-mode!** Big no-no
